pkg/reconciler/knative/resources/knative_configuration_test.go

```diff
@@ -33,14 +33,17 @@ var kfsvc = &v1alpha1.KFService{
 	ObjectMeta: metav1.ObjectMeta{
 		Name:      "mnist",
 		Namespace: "default",
+		Annotations: map[string]string{
+			constants.KFServiceGKEAcceleratorAnnotationKey: "nvidia-tesla-t4",
```

Comment:
Idea here i believe is to extend with annotations from other cloud providers if needed?

Reply:
Absolutely. It's not clear to me if there are additional bits though